datasets/fr/tresor/fr_tresor_gels_avoir.yml

```diff
@@ -56,6 +56,478 @@ assertions:
+          - "décision du comité des sanctions des Nations unies du 16/08/2023 (ONU Corée du Nord - RCSNU 1718 (2006) et R (UE) 2017/1509)"
+          - "Décision du comité des sanctions des Nations Unies du 17/12/2015 (ONU République centrafricaine - RCSNU 2127 (2013) et R (UE) 224/2014)"
+          - "Décision du comité des sanctions des Nations Unies du 20/04/2020 (ONU République centrafricaine - RCSNU 2127 (2013) et R (UE) 224/2014)"
+          - "Décision du comité des sanctions des Nations Unies du 20/08/2015 (ONU République centrafricaine - RCSNU 2127 (2013) et R (UE) 224/2014)"
+          - "décision du comité des sanctions des Nations unies du 23/02/2001 (ONU Afghanistan/Taliban - Résolution 1988 (2011) du CSNU et R (UE) 753/2011)"
+          - "Décision du comité des sanctions des Nations Unies du 23/08/2012 (ONU Somalie - RCSNU 751 (1992) et R (UE) 356/2010)"
+          - "Décision du comité des sanctions des Nations Unies du 23/08/2016 (ONU République centrafricaine - RCSNU 2127 (2013) et R (UE) 224/2014)"
+          - "décision du comité des sanctions des Nations unies du 24/09/2014 (ONU Somalie - RCSNU 751 (1992) et R (UE) 356/2010)"
+          - "décision du comité des sanctions des Nations unies du 25/01/2001 (ONU Afghanistan/Taliban - Résolution 1988 (2011) du CSNU et R (UE) 753/2011)"
+          - "Décision du comité des sanctions des Nations Unies du 25/07/2012 (ONU Somalie - RCSNU 751 (1992) et R (UE) 356/2010)"
+          - "décision du comité des sanctions des Nations unies du 26/05/2023 (ONU Somalie - RCSNU 751 (1992) et R (UE) 356/2010)"
+          - "décision du comité des sanctions des Nations unies du 28/11/2011 (ONU République démocratique du Congo - RCSNU 1533 (2004) et R (CE) 1183/2005)"
+          - "décision du comité des sanctions des Nations unies du 29/03/2007 (ONU République démocratique du Congo - RCSNU 1533 (2004) et R (CE) 1183/2005)"
+          - "décision du comité des sanctions des Nations unies du 29/04/2021 (ONU Libye - RCSNU 1970/2011 et R (UE) 2016/44)"
+          - "décision du comité des sanctions des Nations unies du 29/11/2023 (ONU Libye - RCSNU 1970/2011 et R (UE) 2016/44)"
+          - "décision du comité des sanctions des Nations unies du 29/12/2021 (ONU Al Qaeda Etat - Islamique - RCSNU 1267 (1999) et R (UE) 881/2002 )"
+          - "décision du comité des sanctions des Nations unies du 30/03/2018 (ONU Corée du Nord - RCSNU 1718 (2006) et R (UE) 2017/1509)"
+          - "décision du comité des sanctions des Nations unies du 30/06/2014 (ONU République démocratique du Congo - RCSNU 1533 (2004) et R (CE) 1183/2005)"
+          - "décision du comité des sanctions des Nations unies du 30/11/2012 (ONU République démocratique du Congo - RCSNU 1533 (2004) et R (CE) 1183/2005)"
+          - "décision du comité des sanctions des Nations unies du 31/12/2012 (ONU République démocratique du Congo - RCSNU 1533 (2004) et R (CE) 1183/2005)"
+        value: null
```

Comment:
So these all map to other programs which we do have in our database. Should we add a TODO to tackle this?